Flepper.Tests.Unit/QueryBuilder/Commands/OrderByCommandTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Flepper.QueryBuilder;
+using FluentAssertions;
+using Xunit;
+
+namespace Flepper.Tests.Unit.QueryBuilder.Commands
+{
+    [Collection("CommandTests")]
+    public class OrderByCommandTests
+    {
+        [Fact]
+        public void ShouldCreateSelectWithOrderByStatement()
+        {
+             FlepperQueryBuilder.Select("Id", "Name", "Birthday")
+                .From("user")         
+                .OrderBy("Name")
+                .Build()
+                .Trim()
+                .Should()
+                .Be("SELECT [Id],[Name],[Birthday] FROM [user] ORDER BY [Name]");
+        }
+
+        [Fact]
+        public void ShouldCreateSelectWithOrderByDescStatement()
+        {
+            FlepperQueryBuilder.Select("Id", "Name", "Birthday")
+               .From("user")
+               .OrderByDesc("Name", "Birthday")
+               .Build()
+               .Trim()
+               .Should()
+               .Be("SELECT [Id],[Name],[Birthday] FROM [user] ORDER BY [Name],[Birthday] DESC");
+        }
+        //SELECT [Id],[Name],[Birthday] FROM [user] ORDER BY [Name],[Birthday] DESC
+        //SELECT [Id],[Name],[Birthday] FROM [user] ORDER BY [Name] DESC
```

Comment:
Remove those comments

Reply:
Ops! sorry for the comment lines between the code!